### PR TITLE
fix: bug fixing

### PR DESCRIPTION
**Pull request by juliswer**

### CODE REVIEW

File: src/containers/hero/index.tsx [hunk 1]
```diff
@@ -1,62 +1,85 @@
...
+              >
+                LEDA | NFT Marketplace
+              </span>
+              <h2
+                className="title-hero"
+                data-sal="slide-up"
+                data-sal-delay={200}
+                data-sal-duration={800}
+              >
+                Search your rare NFTs by world <br /> class artists
```
Comment (GimerStackIt) [inline src/containers/hero/index.tsx:38]: Do you mean "Search rare NFTs created by world class artists!" ?

File: src/data/homepages/home-01.json [hunk 2]
```diff
@@ -54,10 +54,10 @@
         },
         {
           "id": 3,
-          "title": "Add your NFT's",
+          "title": "Create your NFTs",
           "path": "/create",
           "subtitle": "Step-02",
-          "description": "We've made the template fully responsive, so it looks great on all devices: desktop, tablets and.",
+          "description": "We've made the template fully responsive, so it looks great on all devices: desktop, tablets and smartphone.",
```
Comment (GimerStackIt) [inline src/data/homepages/home-01.json:60]: I think is better the plural: smartphones

File: src/pages/create.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import withAuth from '@components/auth/withAuth';
 
 const Create = () => (
   <>
-    <SEO pageTitle="Create New" />
+    <SEO pageTitle="Create an NFT" />
     <Breadcrumb pageTitle="Create new NFT" />
```
Comment (GimerStackIt) [inline src/pages/create.tsx:9]: Should it be the same message? Create an NFT?

**Review by GimerStackIt — APPROVED** [review]

**Review by AlefrankM — APPROVED** [review]
LGTM

File: src/containers/create-new/index.tsx [hunk 4]
```diff
@@ -129,7 +127,7 @@ const CreateNewArea = ({ className, space }: Props) => {
                         <i className="feather-upload" />
                         <span className="text-center">Choose a File</span>
                         <p className="text-center mt--10">
-                          PNG, GIF, WEBP, MP4 or MP3. <br /> Max 1Gb.
+                          PNG, JPG or JPEG <br /> Max 1Gb.
```
Comment (mmiranda-stackit) [inline src/containers/create-new/index.tsx:130]: GIF is also supported

File: src/containers/hero/index.tsx [hunk 5]
```diff
@@ -1,62 +1,85 @@
...
-          {homeSection?.headings[0]?.content && (
-            <h2 className="title" data-sal-delay="200" data-sal="slide-up" data-sal-duration="800">
-              {homeSection.headings[0].content}
-            </h2>
-          )}
-          {homeSection?.texts?.map((text: Content) => (
-            <p
-              className="slide-disc"
+
+const HeroArea = ({ homeSection }: Props) => {
```
Comment (mmiranda-stackit) [inline src/containers/hero/index.tsx:11]: Let's change the name of the file and component. Remove Hero

File: src/data/homepages/home-01.json (hunk 2 above)
Comment (mmiranda-stackit) [inline src/data/homepages/home-01.json:60]: Where is this message used?

File: src/features/marketplace/store/marketplace.actions.ts [hunk 6]
```diff
@@ -141,6 +143,7 @@ export const buyItem = createAsyncThunk(
 
       dispatch(openToastSuccess('The NFT has been bought successfully'));
```
Comment (mmiranda-stackit) [inline src/features/marketplace/store/marketplace.actions.ts:144]: imho, we could rephrase this message for something like 'The NFT was successfully purchased.' idk, I find 'bought' not so natural. Thoughts?

File: src/pages/create.tsx (hunk 3 above)
Comment (juliswer) [reply src/pages/create.tsx:9]: It's not required. You can set a message on the SEO (window title) and other on the Breadcrumb

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**